templates/zombienet/src/lib.rs

```diff
@@ -0,0 +1,103 @@
+use zombienet_sdk::{NetworkConfigBuilder, NetworkConfig};
```

Comment:
Now we have tons of boiler plate code. I mean people already copy enough code. Maybe zombienet itself could be extended to check if stuff is part of the `PATH` and throw a proper error. Or how does the error look like right now?

Reply:
Yes, the actual error in zombienet came from the `os` but we can have this check built-in for `native` and remove all this code. 

```
---- parachain_template_block_production_test stdout ----
Error: Provider error

Caused by:
    Error running command 'parachain-template-node --help' in pod parachain-template-node: No such file or directory (os error 2)
```

Reply:
At least telling the user that the searched binary is not in path would be a better UX. Ty

Reply:
Yes, I will add the sanity check as part of zombienet and remove all this code from here 👍.
Thx again for the feedback!